x-pack/plugins/serverless_observability/public/components/side_navigation/index.tsx

```diff
@@ -94,6 +94,10 @@ const navigationTree: NavigationTreeDefinition = {
         },
       ],
     },
+    {
+      type: 'navGroup',
```

Comment:
In the recent mockups, we won't use the ml presets for the observability side nav but instead a custom node tree. It would be great if you revert this change.

Reply:
I've removed this ML section https://github.com/elastic/kibana/pull/159433/commits/3a7eae13415e0a6f36583eacf1b6a6929038baca

It's worth pointing out for other reviews that until these custom ML links are added, there will be no way to access ML pages from the side navigation menu when running the Observability project.
It will still be possible to access ML pages from kibana's app search feature at the top of kibana.